### PR TITLE
`DOMString` -> `string`

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Continuing #15499 

### Summary
`DOMString` is not exposed by browsers to JavaScript. It is converted to `string`.[[ref](https://developer.mozilla.org/en-US/docs/Web/API/DOMString)]

Same is the case with `USVString` and `CSSOMString`.

### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/dragevent/dragevent/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ new DragEvent(type, DragEventInit);
 ### Arguments
 
 - _type_
-  - : Is a `{{domxref("DOMString")}}` representing the name of the event (see
+  - : Is a `string` representing the name of the event (see
```
Comment (teoli2003) [inline files/en-us/web/api/dragevent/dragevent/index.md:33]:
```suggestion
  - : Is a string representing the name of the event (see
```

File: files/en-us/web/api/dragevent/dragevent/index.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ new DragEvent(type, DragEventInit);
 ### Arguments
```
Comment (teoli2003) [inline files/en-us/web/api/dragevent/dragevent/index.md:30]: Drive-by fix:
```suggestion
### Parameters
```

Can you remove the `;` in the syntax above and mark DragEventInit as optional?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Just a couple of small additional fixes.

**Comment by OnkarRuikar** [comment]
done!

**Review by teoli2003 — APPROVED** [review]
